docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md

```diff
@@ -0,0 +1,59 @@
+# CREATE INDEX
+
+## Syntax
+
+```sql
+CreateIndextmt ::=
+    'CREATE' 'INDEX' IndexName ON TableName IndexColumn OptOptionsList
+
+IndexName ::= Identifier
+
+TableName ::=
+    Identifier ('.' Identifier)?
+
+
+IndexColumn ::=
+    IndexColumnPrefix ")"
+
+IndexColumnPrefix ::=
+    "(" ColumnExpression
+    | IndexColumnPrefix "," ColumnExpression
+
+ColumnExpression ::=
+    Identifier
+     
+OptOptionsList ::=
+    "OPTIONS" OptionList
+
+OptionList ::=
+    OptionsListPrefix ")"
+
+OptionsListPrefix ::=
+    "(" OptionEntry
+    | OptionsListPrefix "," OptionEntry
+
+OptionEntry ::=
+    Identifier "=" Identifier
+
+```
+
+**Description**
+
+The `CREATE INDEX` statement is used to create a new index on OpenMLDB. If there are data in the table, some asynchronous jobs will be executed to load data. We can see the job status in `ns_client` with [`showopstatus`](../../../maintain/cli.md) command.
```

Comment:
some asynchronous jobs will be executed to load data
=>
data will be loaded asynchronously